src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,89 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
 import './FindMovie.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+import { getMovie } from '../../api';
+
+const noPreview = 'https://via.placeholder.com/360x270.png?text=no%20preview';
+
+interface Props {
+  movies: Movie[];
+  setMovies: (movies: Movie[]) => void;
+}
+
+export const FindMovie: React.FC<Props> = ({ movies, setMovies }) => {
+  const [query, setQuery] = useState('');
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [notFoundMovie, setNotFoundMovie] = useState(false);
+
+  const findMovie = async () => {
+    try {
+      setIsLoading(true);
+
```

Comment:
don't send a request if you have an invalid `title`, just with spaces, for example
Also - put an error like `Put the movie name`